3rdparty/python/requirements.txt

```diff
@@ -17,8 +15,9 @@ psutil==5.4.8
 Pygments==2.3.1
 pyopenssl==17.3.0
 pystache==0.5.3
-pytest-cov>=2.5,<2.6
-pytest>=3.4,<4.0
+pytest-cov>=2.6,<2.8
```

Comment:
This really needs to happen here as well / more importantly:
https://github.com/pantsbuild/pants/blob/b4e316f4badc2703cea566267886d8b094d0f569/src/python/pants/backend/python/subsystems/pytest.py#L13-L32

I just noticed though that these direct deps are no longer used by pants - yay!:
```
$ git grep 3rdparty/python:pytest
$
```

So please just kill these pytest and pytest-cov requirements outright.

When you do fix `src/python/pants/backend/python/subsystems/pytest.py` you may hit test failures due to #6282. If not, excellent - please add a "Fixes #6282" to the PR description so that issue closes out. If so, then this is not so simple :/